internal/services/securitycenter/security_center_setting_resource.go

```diff
@@ -44,6 +43,8 @@ func resourceSecurityCenterSetting() *pluginsdk.Resource {
 				ValidateFunc: validation.StringInSlice([]string{
 					"MCAS",
 					"WDATP",
+					"WDATP_EXCLUDE_LINUX_PUBLIC_PREVIEW",
```

Comment:
presumably this one will be removed in time?

Reply:
I'm not suggesting we remove this necessarily, fwiw - more a question than anything?

Reply:
I'm not sure, but per the [CLI document](https://learn.microsoft.com/en-us/cli/azure/security/setting?view=azure-cli-latest#az-security-setting-list-examples), it is still supported. Also it shows up in my subscription when I list the settings. Based on this, shall I add it back?